simulation/halsim_gui/src/main/native/include/HALSimGui.h

```diff
@@ -25,4 +28,9 @@ class HALSimGui {
   static std::unique_ptr<glass::NetworkTablesProvider> ntProvider;
 };
 
+WPILIB_DLLEXPORT void AddGuiInit(std::function<void()> initialize);
+WPILIB_DLLEXPORT void AddGuiLateExecute(std::function<void()> execute);
+WPILIB_DLLEXPORT void AddGuiEarlyExecute(std::function<void()> execute);
+WPILIB_DLLEXPORT void GuiExit();
```

Comment:
Based on the discussion in Slack, these should be exported by default, so you can remove the DLLEXPORT and SymbolExports include.